hardware/asic/skywater/Makefile

```diff
@@ -1,46 +1,47 @@
 ROOT_DIR=../../..
-
+include ../asic.mk
 OPENLANE_DESIGNS:=$(OPENLANE_HOME)/designs
 IMAGE_NAME ?= efabless/openlane:2021.07.29_04.49.46 #latest
 ASIC_NODE=skywater
-#defmacro := -D
+USE_SPRAM = 1
```

Comment:
You don't need these 3 variables here since you include asic.mk file, that includes hardware.mk, that includes config.mk file. They are already set at this point.

Reply:
Sorry but USE_SPRAM was reverted to its original value after you and jose said me to set any value to this in my Makefile this is what I did as you both said

Reply:
I need USE_SPRAM =1 and in config,mk it is 0 by default then where I can set them? other than Makefile

Reply:
You can write the following command in the terminal 'make asic-synth USE_SPRAM=1'.